### PR TITLE
PIM-4082: Fix translation keys during job creation

### PR DESCRIPTION
**Pull request by willy-ahva**
| Q                    | A
| -------------------- | ---
| Bug fix?             | yes
| New feature?         | n
| BC breaks?           | no
| CI currently passes? | y
| Tests pass?          | y
| Scenarios pass?      | y
| Checkstyle issues?*  | n
| PMD issues?**        | n
| Changelog updated?   | y
| Fixed tickets        | PIM-4082
| DB schema updated?   | n
| Migration script?    |
| Doc PR               |

### CODE REVIEW

File: src/Pim/Bundle/ImportExportBundle/Validator/Constraints/JobInstance.php [hunk 1]
```diff
@@ -17,7 +17,7 @@ class JobInstance extends Constraint
     /**
      * @var string
      */
-    public $message = 'Failed to create an {{ job_type }} with an unknown job definition';
+    public $message = 'pim_import_export.job_instance.unknown_job_definition';
```
Comment (solivier) [inline src/Pim/Bundle/ImportExportBundle/Validator/Constraints/JobInstance.php]: real question, how are we displaying the job type now ?

Comment (willy-ahva) [inline src/Pim/Bundle/ImportExportBundle/Validator/Constraints/JobInstance.php]: In the job instance validator :)

File: src/Pim/Bundle/ImportExportBundle/Resources/translations/validators.en.yml [hunk 2]
```diff
@@ -1,2 +1,4 @@
 This directory is not writable: This directory is not writable.
 This directory is not valid:    This directory is not valid.
+pim_import_export.job_instance.unknown_job_definition: Failed to create an %job_type% with an unknown job definition
```
Comment (jjanvier) [inline src/Pim/Bundle/ImportExportBundle/Resources/translations/validators.en.yml]: a instead of an ? no ?

**Comment by jjanvier** [comment]
GTM for me

File: spec/Pim/Bundle/ImportExportBundle/Validator/Constraints/JobInstanceSpec.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    function it_returns_the_name_of_the_class_that_validates_this_constraint()
+    {
+        $this->validatedBy()->shouldReturn('pim_job_instance_validator');
+    }
+
+    function it_returns_constraint_targets()
+    {
+        $this->getTargets()->shouldReturn('class');
+    }
+}
```
Comment (filipsalpe) [inline spec/Pim/Bundle/ImportExportBundle/Validator/Constraints/JobInstanceSpec.php]: What's the relation between this modification and the commit message?

Comment (willy-ahva) [inline spec/Pim/Bundle/ImportExportBundle/Validator/Constraints/JobInstanceSpec.php]: Units were broken. I replaced them by specs ;)

**Pull request merged**